includes/editor-templates/panel-elements.php

```diff
@@ -42,7 +42,9 @@
 	<?php if ( ! $has_pro ) : ?>
 	<div id="elementor-panel-get-pro-elements-sticky">
 		<img class="elementor-nerd-box-icon" src="<?php echo ELEMENTOR_ASSETS_URL . 'images/unlock-sticky.svg'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>" loading="lazy" alt="<?php echo esc_attr__( 'Upgrade', 'elementor' ); ?>"/>
-		<div class="elementor-get-pro-sticky-message"><?php echo esc_html( $promotion_data_sticky['message'] ); ?><a target="_blank" href="<?php echo esc_url( $promotion_data_sticky['url'] ); ?>"><?php echo esc_html( $promotion_data_sticky['button_text'] ); ?></a></div>
+		<div class="elementor-get-pro-sticky-message">
```

Comment:
Did we have to change this structure? Or has this been done for readability?

Reply:
If this was done for readability, the closing `</div>` should also be on the next line.